spec/features/audit/lists/filter_spec.rb

```diff
@@ -1,259 +1,461 @@
 RSpec.feature "Filter Content Items to Audit", type: :feature do
-  let!(:me) do
+  scenario "List filters by my unaudited content by default" do
+    given_unaudited_content
+    when_viewing_content_to_audit
+    and_filtering_to_unaudited_content_allocated_to_me_by_default
+    then_the_filtered_list_shows_content_allocated_to_me
+    and_unaudited_content_not_allocated_to_me_is_not_shown
+  end
+
+  scenario "filtering audited content" do
+    given_content_belonging_to_departments
+    when_viewing_content_to_audit
+    and_filtering_to_audited_content_allocated_to_anyone
+    then_the_filter_list_shows_audited_item
```

Comment:
What's the difference between a "filter list" and a "list" and between an "item" (I think that should be "items") and "content"?